keyboards/arrowmechanics/wings/info.json

```diff
@@ -0,0 +1,286 @@
+{
+    "manufacturer": "Arrow Mechanics",
+    "keyboard_name": "WINGS Mark 1",
+    "maintainer": "arrowmechanics",
+    "bootloader": "rp2040",
+    "diode_direction": "ROW2COL",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP6", "pin_b": "GP7"},
+            {"pin_a": "GP6", "pin_b": "GP7"}
+        ]
+    },
+    "matrix_pins": {
```

Comment:
```suggestion
    "features": {
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true,
        "audio": true,
        "encoder": true,
        "rgb_matrix": true
    },
    "matrix_pins": {
```

Reply:
applied, but we don't want to enable bootmagic by default, hence set to false

Reply:
We prefer it to be enabled, as it keeps things consistent and helps when there are issues and a user needs to be told how to perform an eeprom reset.

Reply:
allright, keeping it then